dcpy/connectors/edm/publishing.py

```diff
@@ -182,7 +182,7 @@ def publish(
     By default, keeps draft output folder"""
     if publishing_version is None:
         publishing_version = get_version(draft_key)
-    source = draft_key.path
+    source = draft_key.path + "/"
```

Comment:
This maybe isn't quite ideal but was easiest one line fix - there are a couple things that rely on the current path not including the slash. I think longer term solution would be to rigorously use `Path`s or `PurePath`s for S3 paths, I think it'll just make things a lot saner.